src/App.tsx

```diff
@@ -0,0 +1,187 @@
+                        title={'React'}
+                        body={
+                            'Our React component library can be used for applications built using React v16+. They include strong type definitions so they will work well in either JavaScript or TypeScript projects.'
+                        }
+                        packageName={'@pxblue/react-components'}
+                        url={'/react'}
+                    />
+                    <SimpleCard
+                        image={reactNativeImage}
+                        title={'React Native'}
+                        body={
+                            'Our React Native component library can be used for mobile applications built with React Native v0.60+. They include strong type definitions so they will work well in either JavaScript or TypeScript projects.'
+                        }
+                        packageName={'@pxblue/react-native-components'}
+                        url={'/react-native'}
+                    />
+                </div>
+                <div className={classes.section}>
+                    <div className={classes.sectionContentContainer}>
+                        <Typography className={classes.sectionTitle} variant={'h6'}>
+                            About
+                        </Typography>
+                        <Typography>
+                            Power Xpert Blue is a complete design system for front-end development at Eaton. This system
+                            will help your team build beautiful applications that adhere to our design guidelines while
+                            ensuring flexibility and code reusability.
+                        </Typography>
+                        <div
+                            className={classes.sectionButtonContainer}
+                            style={matchesXS ? { flexDirection: 'column' } : undefined}
+                        >
+                            <Button
+                                className={matchesXS ? classes.sectionButtonXS : classes.sectionButton}
+                                variant="outlined"
+                                color="primary"
+                            >
+                                release plan
+                            </Button>
+                            <Button
+                                className={matchesXS ? classes.sectionButtonXS : classes.sectionButton}
+                                variant="outlined"
+                                color="primary"
+                                href="https://pxblue.github.io/get-started/new-project"
+                            >
+                                choose a framework
+                            </Button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <footer className={classes.footer}>Copyright 2020 Eaton. Licensed under BSD-3-Clause.</footer>
```

Comment:
Can we make this sticky to the bottom of the page?

Reply:
Resolve: after discussion, we understand this is not an issue